lib/src/countly_flutter.dart

```diff
@@ -592,13 +618,17 @@ class Countly {
     return result;
   }
 
+  /// Does nothing
```

Comment:
Sure it does nothing but what was it for? Something related to stopping a session. Then we can say this is deprecated and now will do nothing